pcdsdevices/interface.py

```diff
@@ -524,14 +525,23 @@ def __init__(self, *args, **kwargs):
         self._last_status.set_finished()
         super().__init__(*args, **kwargs)
 
+    def _log_move_limit_error(self, position, ex):
+        logger.error('Failed to move %s from %s to %s: %s', self.name,
+                     self.wm(), position, ex)
+
     def _log_move(self, position):
         logger.info('Moving %s from %s to %s', self.name, self.wm(), position)
 
     def _log_move_end(self):
         logger.info('%s reached position %s', self.name, self.wm())
 
-    def move(self, *args, **kwargs):
-        st = super().move(*args, **kwargs)
+    def move(self, position, *args, **kwargs):
+        try:
+            st = super().move(position, *args, **kwargs)
+        except ophyd.utils.LimitError as ex:
+            self._log_move_limit_error(position, ex)
+            raise
```

Comment:
This will cause `bluesky` scans to miss the `LimitError`, since for subclasses of `PositionerBase`, `set` and `move` are the same thing: https://github.com/bluesky/ophyd/blob/4440bc74f226fccac6bb59a17753b9c517cbe864/ophyd/positioner.py#L46-L88